packages/react-keytips/src/docs/Spec.md

```diff
@@ -0,0 +1,190 @@
+| `positioning`  | `PositioningProps`          | `{ align: 'center', position: 'below' }` | Positioning props to be passed to Keytip.                                                                                                                         |
+| `onExecute`    | `ExecuteKeytipEventHandler` |                                          | Callback function triggered when keytip is activated.                                                                                                             |
+| `onReturn`     | `ReturnKeytipEventHandler`  |                                          | Callback function triggered when the return sequence is pressed.                                                                                                  |
+| `keySequences` | `string[]`                  |                                          | Array of KeySequences which is the full key sequence to trigger this keytip. Should not include initial 'start' key sequence.                                     |
+| `dynamic`      | `boolean`                   |                                          | Whether or not this keytip will have dynamic content: children keytips that are dynamically created (DOM is generated on keytip activation), Menu, Tabs or Modal. |
+| `appearance`   | `normal \| inverted`        | `inverted`                               | Visual appearance of Keytip.                                                                                                                                      |
+| `visible`      | `boolean`                   | `false`                                  | Control the Keytip's visibility programmatically.                                                                                                                 |
+| `content`      | `string`                    |                                          | The text content of the Keytip.                                                                                                                                   |
+
+## Structure
+
+### JSX Tree
+
+```tsx
+<Portal>
+  <Keytip content="A" positioning={{ target }} />
+  <Keytip content="B" positioning={{ target }} />
+  <Keytip content="C" positioning={{ target }} />
+</Portal>
+```
+
+### DOM
+
+```html
+<!-- Portal from Keytips -->
+<div>
+  <div role="tooltip" id="tooltip-1" class="{tooltip}" data-popper-placement="bottom">A</div>
+  <div role="tooltip" id="tooltip-2" class="{tooltip}" data-popper-placement="bottom">B</div>
+  <div role="tooltip" id="tooltip-3" class="{tooltip}" data-popper-placement="bottom">C</div>
+</div>
+```
+
+## Migration
+
+See [MIGRATION.md](./MIGRATION.md).
+
+## Behaviors
+
+## Entering and Exiting Keytips Mode
+
+- When the `startSequence` is pressed, the keytips component will enter keytips mode. By default the `startSequence` is `'alt+control'` (option+control on macOS).
```

Comment:
```suggestion
- When the `startSequence` is pressed, the keytips component will enter keytips mode. By default the `startSequence` is `'alt+Windows'` (option+control on macOS).
```

Reply:
@ValentinaKozlova I guess the suggestion is not correct, it should be alt+control / option-control, `win` key is different one.

Reply:
In the design spec it says Windows, and in V8 on Windows keytips are triggered by `alt+windows` not by `alt+control`

Reply:
right, good catch then :) Updated